src/core/server/http/lifecycle/auth.test.ts

```diff
@@ -23,7 +23,7 @@ import { adoptToHapiAuthFormat } from './auth';
 const SessionStorageMock = {
   asScoped: () => null as any,
 };
-const requestMock = {} as any;
+const requestMock = { route: { settings: {} } } as any;
```

Comment:
__note:__ I'm surprised we don't have a common internal Hapi request mock in core's `http` module yet :D

Reply:
will add